gem/models.py

```diff
@@ -15,6 +17,28 @@
 from wagtail.images.edit_handlers import ImageChooserPanel
 
 
+class VideosPage(Page):
+    date = models.DateField("Post date")
+    intro = models.CharField(max_length=250)
+    body = RichTextField(blank=True)
+    videourl = models.CharField(max_length=250, null=True)
```

Comment:
Django has a URLField https://docs.djangoproject.com/en/1.11/ref/models/fields/#urlfield

(Just a note, we use Django 1.11.x)